keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -193,7 +198,8 @@ updates.
 Currently, there is a lack of a standardized approach to define a
 cluster inventory. However, with the growing number of users managing
 multiple clusters and deploying applications across them, projects like
-Open Cluster Management (OCM), Karmada, Clusternet, and Fleet Manager
+[Open Cluster Management (OCM)](https://open-cluster-management.io/),
+[Clusternet](https://clusternet.io/) or [Kubernetes Fleet Manager](https://github.com/Azure/fleet)
```

Comment:
Hi @ryanzhang-oss ,
It doesn't make sense to move Karmada out of the list.

Reply:
I think it needs to be added, we even mention karmada at the kubecon talk about this API